android/src/main/java/io/sentry/RNSentryModule.java

```diff
@@ -146,6 +146,11 @@ public void captureEnvelope(String envelope, Promise promise) {
         promise.resolve(true);
     }
 
+    @ReactMethod
+    public void getStringBytesLength(String str) {
+        return string.getBytes(str)
```

Comment:
I think it would be something like `str.getBytes("UTF-8").length`

Reply:
string, `string` is also C# btw :)

Reply:
And the build passed lol

Reply:
lol, true